src/Compilers/CSharp/Portable/FlowAnalysis/PreciseAbstractFlowPass.cs

```diff
@@ -311,7 +312,7 @@ protected override bool ConvertInsufficientExecutionStackExceptionToCancelledByS
 
         /// <summary>
         /// A pending branch.  There are created for a return, break, continue, goto statement,
-        /// yield return, yield break, await expression, and if PreciseAbstractFlowPass.trackExceptions
+        /// yield return, yield break, await expression, foreach/using await, and if PreciseAbstractFlowPass.trackExceptions
```

Comment:
Are there pending branches for `foreach`?

Reply:
Not yet, but that will be coming shortly with foreach-await. I updated the comment pro-actively, as I feared I'd forget to update it in the foreach-await PR. I can replace with PROTOTYPE if you prefer.

---
In reply to: [160547396](https://github.com/dotnet/roslyn/pull/23961#discussion_r160547396) [](ancestors = 160547396)